src/scss/theme/_image-cta.scss

```diff
@@ -58,6 +70,24 @@
 // Then each card is selected by position.
 // Also sets the black hover color on the text link.
 .su-page-components {
+
+  // Setting up the external link icon to be white on hover all types.
+  > div {
+    div {
+      &.ptype-stanford-image-cta {
+        .su-image-cta-paragraph {
+          &:hover {
+            .su-link--external {
+              &::after {
+                background-color: $su-color-white;
```

Comment:
Do you need the level of specificity?

Reply:
Yesterday I couldn't get it to work without it, but I will look and try again. Maybe a fresh look will help.

Reply:
Woo, I made it less specific and it is working still.  Thanks for the prompting.

Reply:
This warms the cockles of my heart.